Client/src/Components/Snackbar/MySnackbar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Alert, Snackbar } from "@mui/material";
+export default function MySnackbar(props) {
+  const { vertical, horizontal, open, severity, message, handleClose } =
+    props.state;
+
+  return (
+    <Snackbar
+      open={open}
+      autoHideDuration={3000}
+      anchorOrigin={{ vertical, horizontal }}
+      key={vertical + horizontal}
```

Comment:
quite strange key :) seems `vertical + horizontal` doesn't guarantee a unique string. However, seems you don't need a key here at all, Snackbar is rendered just once, it's not a list of them

Reply:
I saw such key in example on MaterialUI and just copied it, I remove it and if everything will work correct, I don't return it

Reply:
probably in example they had several Snackbar components with different vertical and horizontal properties combinations under one parent, only in that case it makes sense